doc/changes.rst

```diff
@@ -1,6 +1,79 @@
 Changelog
 =========
 
+Release v0.2.1 (2020/XX/XX)
+---------------------------
+
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4025418.svg
+   :alt: Digital Object Identifier for the Zenodo archive
+   :target: https://doi.org/10.5281/zenodo.4025418
```

Comment:
Need to update the DOI.

Reply:
Zenodo DOI has been reserved for PyGMT v0.2.1. https://doi.org/10.5281/zenodo.4253459 (will work once we publish the release).

```suggestion
.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4253459.svg
   :alt: Digital Object Identifier for the Zenodo archive
   :target: https://doi.org/10.5281/zenodo.4253459
```

Reply:
Btw, the BibTeX needs to be updated with the new Zenodo DOI too.